sotodlib/coords/optics.py

```diff
@@ -330,7 +330,7 @@ def LAT_optics(zemax_path):
     x = LAT["x"][gi].ravel()
     y = LAT["y"][gi].ravel()
     xi = LAT["elev"][gi].ravel()
-    eta = LAT["eta"][gi].ravel()
+    eta = LAT["xel"][gi].ravel()

```

Comment:
Are we making an approximation here when converting "elev, xel" into "xi, eta"?  If so ... does that matter?  Elevation and cross-elevation are similar to xi, eta, if you're near elevation 0... but when you get a few degrees away from that origin, the difference starts to matter.

Reply:
The zemax model defines the origin to be at the boresite (since it doesn't know anything about sky coordinates and is modeled at a fixed configuration). So I think in this case el/xel is the same as xi/eta. So the only elevation dependent effect here is the LOS rotation. 

I think even if it was actual el/xel then it would still be OK since we could just compute the template at el=0 and rotate to get the correct orientation (which is basically what happens right now anyways).

Reply:
The point is that "el, xel" is not the same as "xi, eta", even stipulating that el=0.  The coordinate conversion to get (xi, eta) from (el, xel) is easy to do with so3g.proj.quat functions -- I guess I am proposing that we do that properly instead of ignoring the difference.

Reply:
> I suppose el/xel is actually a lonlat pair? 

That is the concern.  Is it?  If so, you'll want to do something like (depending on the sign convention for xel):
```
xi, eta, _ = quat.decompose_xieta(quat.euler(1, -np.pi/2) * quat.rotation_lonlat(-xel, el))
```
That takes, for example,
```
>>> el, xel = .01, .02
>>> xi, eta, _ = quat.decompose_xieta(quat.euler(1, -np.pi/2) * quat.rotation_lonlat(-xel, el))
>>> xi, eta
(0.019997666768331163, 0.009999833334166508)
```

Though it's also confusing to me that you associate (xi, eta) with (el, xel) ... rather than (xel, el).  That will need to be sorted.  Draw me a picture and we'll work it out...